frontend/dockerfile/dockerfile_lint_test.go

```diff
@@ -233,30 +296,35 @@ COPY $bar .
 
 func testRuleCheckOption(t *testing.T, sb integration.Sandbox) {
 	dockerfile := []byte(`#check=skip=all
+#
```

Comment:
I don't think this should be needed. Directives are known comments and not associated with `FROM`. Probably makes sense to just ignore the whole first comment block(until empty line) as it is more likely about the whole file.

Reply:
I think directives section can be taken out in the AST layer so that they do not appear as `PrevComment` in the AST. For the very first stage without global ARG we can maybe just ignore the warning.

Reply:
Done!